src/components/ReportActionItem/MoneyRequestPreview/MoneyRequestPreviewContent.tsx

```diff
@@ -211,6 +211,16 @@ function MoneyRequestPreviewContent({
 
     const displayAmount = isDeleted ? getDisplayDeleteAmountText() : getDisplayAmountText();
 
+    const shouldShowSplitShare = isBillSplit && !!requestAmount && requestAmount > 0;
+    const splitShare = useMemo(
+        () =>
+            shouldShowSplitShare &&
+            (transaction?.comment?.splits
+                ? transaction?.comment?.splits?.find((split) => split.accountID === action.actorAccountID)?.amount
```

Comment:
As of now I see equal share for the other user [as expected in this comment](https://github.com/Expensify/App/pull/40386#issuecomment-2071234655).

But here the actorAccountID for the action is the one who creates the split, right? I think we should use here currentUserAccountID.

Reply:
Totally